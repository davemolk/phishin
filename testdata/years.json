{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":[{"date":"1983-1987","show_count":34},{"date":"1988","show_count":44},{"date":"1989","show_count":64}]}