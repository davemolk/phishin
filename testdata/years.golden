Years:     Show Count:
1983-1987  34
1988       44
1989       64
